<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TexturesWidget</class>
 <widget class="QWidget" name="TexturesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="textureSelectionLayout" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="texturesListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="buttonsLayout" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="textureNameLineEdit"/>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="TexturePreview" name="texturePreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>144</width>
       <height>144</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="textureSettingsWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="typeComboBox">
        <item>
         <property name="text">
          <string>Image Texture</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Color Texture</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ColorButton" name="colorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="FileSelector" name="pathWidget" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="flipVerticallyLabel">
        <property name="text">
         <string>Flip Vertically </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="flipVerticallyCheckBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="colorSpaceLabel">
        <property name="text">
         <string>Color Space</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="colorSpaceComboBox">
        <item>
         <property name="text">
          <string>Linear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sRGB</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="filteringLabel">
        <property name="text">
         <string>Filtering </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="filteringComboBox">
        <item>
         <property name="text">
          <string>Bilinear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nearest</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="wrappingLabel">
        <property name="text">
         <string>Wrapping </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="wrappingComboBox">
        <item>
         <property name="text">
          <string>Repeat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mirrored repeat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clamp to edge</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clamp to border</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="borderColorLabel">
        <property name="text">
         <string>Border Color</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="ColorButton" name="borderColorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>ColorButton.hpp</header>
  </customwidget>
  <customwidget>
   <class>FileSelector</class>
   <extends>QWidget</extends>
   <header>FileSelector.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TexturePreview</class>
   <extends>QLabel</extends>
   <header>TexturePreview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
